test/Tester/OrleansLoggingTests.cs

```diff
@@ -23,10 +24,10 @@ public void OrleansLoggingCanConfigurePerCategoryServeriyOverrides()
             IServiceCollection serviceCollection = new ServiceCollection();
             serviceCollection.AddLogging();
             var severityOverrides = new OrleansLoggerSeverityOverrides();
-            severityOverrides.LoggerSeverityOverrides.TryAdd(this.GetType().FullName, Severity.Warning);
-            serviceCollection.AddLogging(builder => builder.AddOrleansLogging(new List<ILogConsumer>()
+            severityOverrides.LoggerSeverityOverrides.Add(this.GetType().FullName, Severity.Warning);
+            serviceCollection.AddLogging(builder => builder.AddLegacyOrleansLogging(new List<ILogConsumer>()
             {
-                new FileLogConsumer($"{this.GetType().Name}.log")
+                new LegacyFileLogConsumer($"{this.GetType().Name}.log")
             }, severityOverrides));
             var serviceProvider = serviceCollection.BuildServiceProvider();
```

Comment:
I meant that you do:
```
using (var serviceProvider = serviceCollection.BuildServiceProvider())
{
...
}
```
and then do not do not use Dispose explicitly. But on hindsight, BuildServiceProvider returns IServiceProvider which is not itself disposable, the concrete implementation is, so you might not be able to do it

Reply:
no I cannot do that since serviceProvider is not disposable. you want me to revert to original line?

Reply:
Yes